install.py

```diff
@@ -21,9 +14,9 @@ def check_sim_package_exists() -> bool:
         return False
 
 
-def sim_setup():
+def sim_setup(sim_path: Union[str, Path], setup_filepath: Union[str, Path]):
     if check_sim_package_exists():
-        print(f'pySimPulseqSBB already installed. You can start your simulations.')
+        print(f'pySimPulseqSBB already installed. You can start ypipour simulations.')
```

Comment:
```suggestion
        print(f'pySimPulseqSBB already installed. You can start your simulations.')
```
That was the wrong window 😄